src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
-export interface Figure {
 
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+export interface Figure {
+  shape: Shapes,
+  color: Color,
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shapes = 'triangle';
```

Comment:
And use this enum in each of your Figure
```suggestion
  public shape: Shapes = Shapes.Triangle;
```